Transformers/Transformers.ipynb

```diff
@@ -0,0 +1,278 @@
+{
```

Comment:
<p>Soooo ... should data and model downloads be logically separated? Like in separate cells? </p><p> </p><p>We should also use <code>tarfile</code> instead of <code>!tar xvf</code>   I think. I am guessing it'd be a nice pretty util check which you can add to the get_files... function itself. Like if tar file, just decompress it for me please?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NirantK/Hinglish/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Moved this to get_files_from_drive - where it checks if the extension is tar and extracts it directly. <a href="https://github.com/NirantK/Hinglish/commit/1d06aa2c1c80324d69866b1db335a5981bb84a9f" target="_blank">1d06aa2</a></p><p> </p>